eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/listeners/Count.java

```diff
@@ -0,0 +1,67 @@
+package uk.ac.ebi.eva.accession.clustering.batch.listeners;
```

Comment:
Since this class is pretty generic, I am wondering why we shouldn't have this in variation-commons...

Reply:
I discussed this with Andres but we could not decide on where to put the code and so decided to keep it here for the time being and refactor it the next time we are going to touch this (implementing storing count for other process in accession.) But if you have something in mind, we can discuss and refactor.